src/lib/flash-helper.js

```diff
@@ -16,8 +16,9 @@ const ensureError = utilities.ensureError;
 // Flashing an NCP firmware can take a few minutes
 const FLASH_TIMEOUT = 4 * 60000;
 
-async function flashFiles({ device, flashSteps, resetAfterFlash = true, ui }) {
-	const progress = _createFlashProgress({ flashSteps, ui });
+async function flashFiles({ device, flashSteps, resetAfterFlash = true, ui, verbose=true }) {
+	let progress = null;
+	progress = verbose ? _createFlashProgress({ flashSteps, ui, verbose }) : null;
```

Comment:
Ideally, the check for a `verbose` flag should go for every exposed method in this class. I have added the `verbose` flag only to this method because this is the only one impacting the device protection operations during bootloader flashing locally.